File: 100-Days-Series/Day 59/InvertedHalfPyramid.java
class InvertedHalfPyramid{
    // Function to print Inverted Half Pyramid
    public static void getPrintPyramid(int rows){
        for(int i=rows;i>=1;i--){
            for(int j=1;j<=i;j++){
                System.out.print(j+" ");
            }
            System.out.println();
        }
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start Here
        int rows;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of rows : ");
        rows=sc.nextInt();
        getPrintPyramid(rows);
    }
}

File: 100-Days-Series/Day 59/InvertedHalfPyramid.py

# Function to print Inverted Half Pyramid
def get_print_pyramid(rows):
    for i in range(rows,0,-1):
        for j in range(1,i+1,1):
            print(j,end=" ")
        print()

# Let's Start Here
rows=int(input("Enter the number of rows : "))
get_print_pyramid(rows)